Lib/test/test_print.py

```diff
@@ -155,6 +155,41 @@ def test_string_with_excessive_whitespace(self):
 
         self.assertIn('print("Hello World", end=" ")', str(context.exception))
 
+    def test_print_string_with_stream_redirection(self):
+        import sys
+        python2_print_str = 'print >> sys.stderr, "message"'
+        with self.assertRaises(TypeError) as context:
+            exec(python2_print_str)
+
+        self.assertIn('Did you mean "print(<message>, '
+                'file=<output_stream>)', str(context.exception))
+
+    def test_print_with_stream_redirection_using_max(self):
+        import sys
+        python2_print_str = 'max >> sys.stderr'
+        with self.assertRaises(TypeError) as context:
+            exec(python2_print_str)
+
+        self.assertNotIn('Did you mean "print(<message>, '
+                'file=<output_stream>)', str(context.exception))
+
+    def test_print_with_stream_redirection_using_rrshift(self):
+        import sys
+
```

Comment:
Too much empty lines. Use empty lines only for separating different tests in one method.